cluster/containers.go

```diff
@@ -0,0 +1,64 @@
+package cluster
```

Comment:
Also, please grep for `[]*Container` in the project (probably `Engine` and so on) and replace it with `Containers`